python/test/openmldb_client_test.py

```diff
@@ -40,7 +41,8 @@ def test_basic(self):
       pass
 
     time.sleep(2)
-
+    
+    self.check_sqlalchemy_table_api(engine)
     connection.execute(ddl)
```

Comment:
It would be better to add unit test for `has_table` API.

Reply:
updated.